### DISCUSSION TITLE
Passionate about designing green software solutions?

### DISCUSSION THREAD

**Discussion by russelltrow** [Green-Software-Foundation/patterns#309] [Announcements]
**Are you a software practitioner in a Design, Product or Research role and part of the Green Software Foundation (GSF)?** We’re seeking those outside of engineering roles who are passionate about green software and keen to play their part in reducing the carbon emissions of software.

This opportunity is exclusive to employees of member organizations of the Green Software Foundation. [Check if your organization is a member](https://greensoftware.foundation/#steering-members).

## The Opportunity
We are looking for people to participate in a series of informal knowledge sharing sessions to discuss Green Software Design. We wish to provide a forum where interested parties can gather and share their experiences and aspirations.

We believe that potentially there could be enough interest in this area from across the GSF to warrant the creation of a new project which could enable design & product professionals to take practical actions in reducing the carbon emissions of their digital solutions.

## We're seeking individuals who have

* A passion for sustainable software design
* A background in a less engineering-centric areas of software - UX/UI Design, User Research, Product Management, Web Design
* Ability to commit 1-2 hours per month

## We need you to help us
* Understand the needs of a broader range of green software practitioners
* Identify the gaps in the current available resources for designers
* Generate ideas for new GSF green software design projects

## Register your interest

We will be running the first session virtually on **Tuesday 19th March at 14:00 GMT.**

**To get involved simply post a comment below.** [Creating a GitHub account](https://github.com/signup) is free and easy.

# References

* https://github.com/Green-Software-Foundation/community-wg/issues/101
* https://github.com/Green-Software-Foundation/patterns/issues/302

**Comment by russelltrow** [comment]
Hey @pdematos12, @philippqf, @josephrossetto & @cf-almeida!

**Reply by philippqf** [reply]
I'm in 😃

**Reply by pdematos12** [reply]
Looking forward to it.

**Reply by Nimishgi** [reply]
I am in :)

**Reply by pdematos12** [reply]
@russelltrow - in case yo missed @Nimishgi would like to join too.

**Reply by russelltrow** [reply]
Invite send @Nimishgi

**Comment by shashi-aveva** [comment]
Hello!

I'm interested.
(Shashi Prakash, UX designer @ AVEVA from Paula's team.)

**Reply by russelltrow** [reply]
Hi @shashi-aveva, thank you so much for expressing your interest. Could I please ask that you complete https://greensoftware.foundation/onboarding/ so I can get your email and add you to the call.

**Reply by shashi-aveva** [reply]
Done.

**Reply by russelltrow** [reply]
Fantastic thank you. I've added you to the calendar invite.

**Comment by kunKun-tx** [comment]
I'd like to join, IT Engineering Manger @HARC.

**Reply by russelltrow** [reply]
Hi @kunKun-tx, wonderful to have your onboard. I've just added you to the calendar invite.

**Comment by Mettenim** [comment]
I would also join, thanks Minette, Principal Experience Designer @ Thoughtworks

**Reply by russelltrow** [reply]
Hi @Mettenim, wonderful to have your onboard. I've just added you to the calendar invite.

**Comment by ilfff** [comment]
Hi @russelltrow, I'd love to participate

**Reply by russelltrow** [reply]
Hi @ilfff, thank you so much for expressing your interest. Could I please ask that you complete https://greensoftware.foundation/onboarding/ so I can get your email and add you to the call.

**Reply by ilfff** [reply]
Hi @russelltrow I've just realized that the call is just for specific companies' employees. Unfortunately I don't work for any of the suggested companies, is there any chance to participate anyway? 
Thank you,
Have a nice day

**Reply by russelltrow** [reply]
Hi @ilfff unfortunately the call is only for employees of GSF Member Organisations. If interested the joining information is available here: https://greensoftware.foundation/join-us

**Comment by russelltrow** [comment]
A huge thank you to everyone who was able to attend the first session last week.

For those not able to attend below is a brief summary of the activities that took place and next steps.

---

### WHAT DID WE DO?

We started with a round of introductions and got to know a little more about each other.

We then moved onto discussing 3 questions:

* What has brought you here today?
* What challenges are you facing in this space?
* What do you hope to achieve?

We identified a few common themes:
* Need to start by gaining an understanding of the fundamental of green software design
* Measurement is critical - Desire to have evidence to confirm whether a particular approaches or design choice is effective in making software greener
* Want Guidelines - Need resources with established principles and practises for greener design

My favourite comment was when a participant said "I want to be a problem solver not a problem maker"

Everything is on [Miro and available to view](https://miro.com/app/board/uXjVNglVFfI=/?share_link_id=404261555541)

---

### WHAT NEXT?

We finished the session eager to continue the discussion and continue refining the ideas raised.

* We agreed to organise further meetings with a 2 week cadence.
  * The next meeting will be Tuesday 9th April at 14:00 GMT
  * This is to avoid a UK bank holiday
* Everyone will try and recruit more members for the group
  * Please do share the [recruitment page](https://github.com/Green-Software-Foundation/patterns/discussions/309) with your network
* Review the [existing green design resources](https://github.com/Green-Software-Foundation/patterns/issues/302)
* Finally, for the next session we asked everyone to consider a question - "If you had a magic wand that could do anything, how would you use it  for Green Software Design?"

**Comment by russelltrow** [comment]
Next week will be holding our second session on Tuesday 9th April at 13:00 GMT.

Drop a comment here to join us.

**Comment by russelltrow** [comment]
Hi all, our third session be tomorrow - [Tuesday 23rd April at 13:00 GMT](https://everytimezone.com/s/42ec24bd).

Drop a comment here to join us.

**Comment by russelltrow** [comment]
The group will be meeting for the 7th time on Tuesday 18th June, please let me know if you wish to join.

**Comment by russelltrow** [comment]
Hi all, I'm very pleased to introduce @nathalymp a Visual Designer at Avanade who is keen to join the group. I've already shared the calendar invite.

cc: @pdematos12
